test/common/http/conn_manager_impl_test.cc

```diff
@@ -1194,6 +1194,27 @@ TEST_F(HttpConnectionManagerImplTest, RouteShouldUseNormalizedHost) {
   filter_callbacks_.connection_.raiseEvent(Network::ConnectionEvent::RemoteClose);
 }
 
+// Observe host header w/o trailing dot in host when trailing dot removal is configured.
```

Comment:
This makes it sound like the input doesn't have a trailing dot, maybe `Observe tat we strip the trailing dot ...`?

Reply:
thanks, will change.